### ISSUE TITLE
Update Reporting Periods Table and Add Certification Modal

### ISSUE THREAD

**Issue by Vikariusu** [usdigitalresponse/cpf-reporter#415]
## Dependencies
Blocked by #410 (handles backend logic for certification and next reporting period selection)
## Overview
Update the Reporting Periods page to display all reporting periods. Implement functionality to certify the current reporting period and view information about previously certified periods.
## Columns
1. Name - display reporting period's name
2. Start Date (mm/dd/yyyy)
3. End Date (mm/dd/yyyy)
4. Certified At
	1. If certified, this should display: "[Date Certified] by [email of user who certified]" ex: "07/01/2024 by [cvaldivia@usdigitalresponse.org](mailto:cvaldivia@usdigitalresponse.org)"
	2. If Current reporting period, a button should display that says "Certify Reporting Period"
5. Actions:
	1. Display edit button if reporting period is not certified
	2. Empty if reporting period is certified

## Certifying a Reporting Period
If you click the "Certify Reporting Period" button, a window will pop up and ask you to confirm that you want to certify the period:

![Image](https://github.com/user-attachments/assets/4d0d0578-5e10-4ceb-ad3b-f7639ab385cf)

If you select Cancel, the period will not be certified and you will return to the Reporting Periods Page.

If you select "Certify", the following will happen:
- On the reporting period page, the "Certified At" column will update to display: "[Date Certified] by [email of user who certified]" ex: "07/01/2024 by cvaldivia@usdigitalresponse.org"
- [Handled by #413] When users change the Reporting Period toggle to a certified reporting period, they will not see the button for "Submit Workbook" and will have no way to upload files for the certified period (including staff & admin roles)
- [Handled by #410] The next reporting period in terms of next upcoming start date will become the default reporting period and users will be able to upload files for that period. If there is no subsequent reporting period, the latest one will remain the default, but you will not be able to upload files.
- [Handled by #410] Only one reporting period can be open (i.e. accepting upload files) at a time

**Comment by ClaireValdivia** [comment]
@Vikariusu Confirming that templates will not display for users

**Comment by ClaireValdivia** [comment]
@Vikariusu the reporting period page itself looks great, thank you! 
When I try to certify the reporting period, however, I get an error that I am unable to certify, even though a subsequent reporting period exists. I'll create another issue for this error

![Image](https://github.com/user-attachments/assets/ddcfe12d-d3c5-4161-9b94-7c7eb6781904)
